GVFS/GVFS.FunctionalTests/Program.cs

```diff
@@ -60,17 +60,20 @@ public static void Main(string[] args)
 
             if (runner.HasCustomArg("--windows-only"))

```

Comment:
Outside the scope of this PR, but do we need to maintain the ability to run the core tests in the .NET Framework project, if we never run them from that project? Not saying we should necessarily, but it seems like we could simplify this so that the .NET Framework test project contains only the Windows-specific tests. Any reason not to?

Reply:
I can't remember I ran the functional tests from the IDE :-). If that's something you do, I'm totally fine with making it easier.

Running all of the Windows tests on .NET Framework may actually make more sense, since the Windows client is still using .NET Framework. The current setup is also working well enough, so this is a someday-maybe clean up as far as I'm concerned.